docs/vaahcms-2/backend/generate-crud.md

```diff
@@ -50,7 +51,7 @@ Use following procedures to set up CRUD.
 ## Step-4
 Now write migrations according to schema and re-activate module to run the migration.
 Below link you can use to go module in your browser
-```
+```php
 http://127.0.0.1:8000/backend#/vaah/modules/
```

Comment:
@wri-abinash-s001 update `http://127.0.0.1:8000` to `<public-url>` and other places if you have used `http://127.0.0.1:8000`